File: QuanLiNhaHang_C#/QuanLyNhaHang/LoaiBan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyNhaHang
{
    public class LoaiBan
    {
        // Automatic properties
        public string maLoaiBan { get; set; } // primary key
        public string hangSanXuat { get; set; }

        // constructor
        public LoaiBan() { }
        public LoaiBan(LoaiBan lb)
        {
            this.maLoaiBan = lb.maLoaiBan;
            this.hangSanXuat = lb.hangSanXuat;
            this.ds_ban = lb.ds_ban;
        }
        public LoaiBan(string maLoaiBan, string hangSanXuat, ICollection<Ban> ds_ban)
        {
            this.maLoaiBan = maLoaiBan;
            this.hangSanXuat = hangSanXuat;
            this.ds_ban = ds_ban;
        }
        // related = plural
        public ICollection<Ban> ds_ban { get; set; }
    }
}

File: QuanLiNhaHang_Java/src/mypack/LoaiBan.java
package mypack;

import java.util.ArrayList;
import java.util.List;

public class LoaiBan {
	
	// Automatic Properties
    public String maLoaiBan; // Primary Key
    public String hangSanXuat;

    // Constructor
    public LoaiBan() { }
    public LoaiBan(String maLoaiBan, String hangSanXuat, ArrayList<Ban> dsLoaiBan)
    {
        this.maLoaiBan = maLoaiBan;
        this.hangSanXuat = hangSanXuat;
        this.dsLoaiBan = dsLoaiBan;
    }
    public LoaiBan(LoaiBan lb)
    {
        this.maLoaiBan = lb.maLoaiBan;
        this.hangSanXuat = lb.hangSanXuat;
        this.dsLoaiBan = lb.dsLoaiBan;
    }

    // Related = Plural
    public List<Ban> dsLoaiBan;
    
    // Getter - Setter
    public String getmaLoaiBan() {
    	return this.maLoaiBan;
    }
    public void setmaLoaiBan(String maLoaiBan) {
    	this.maLoaiBan = maLoaiBan;
    }
    
    public String gethangSanXuat() {
    	return this.hangSanXuat;
    }
    public void sethangSanXuat(String hangSanXuat) {
    	this.hangSanXuat = hangSanXuat;
    }
    
    public List<Ban> getdsLoaiBan() {
    	return this.dsLoaiBan;
    }
    public void setdsLoaiBan(List<Ban> dsLoaiBan) {
    	this.dsLoaiBan = dsLoaiBan;
    }
}

